src/test/solution_space/solution_space_handler/filter_incorrect_fragments_test.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
+
+import pytest
```

Comment:
Pytest is not a standard library, so please, move this import into a separated second group